src/lib-imap/fuzz-imap-utf7.c

```diff
@@ -0,0 +1,14 @@
+/* Copyright (c) 2020 Dovecot authors, see the included COPYING file */
+
+#include "lib.h"
+#include "fuzzer.h"
+#include "imap-utf7.h"
+
+FUZZ_STR(const char *str)
+{
+	string_t *dest = t_str_new(size);
```

Comment:
use of `size` feels magic because of the macro

Reply:
Oh, right, I was just copying it from the original test. I didn't intend to have/use hidden variables, although they do exist.. Should rename them at least.